.travis.yml

```diff
@@ -5,7 +5,7 @@ branches:
 language: ruby
 cache:
   - bundler
-  directories:
+  - directories:
     - /home/travis/.rvm
```

Comment:
Might need to be:

```
bundler: true
directories:
  - /home/travis/.rvm
```

Reply:
we'll worry about that later. Want green more than I want cache atm